2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,14 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    let total= [];
+    let tot= 0;
+    let sum = 0;
+    for(let element of closingPricesForAllStocks){
+      sum = element.reduce((a, b) => a + b);
+      tot = sum / 5;
```

Comment:
Good use of .reduce() function.
The constant literal 5 is an example of a "magic number" that should be avoided. What would happen to this calculation if the number of prices changed from 5 to any other number?

Reply:
It would be wrong.. Should I use .length? I will try and do it without magic numbers.